lib/storage/data/LocationConstraintParser.js

```diff
@@ -0,0 +1,154 @@
+const https = require('https');
+const http = require('http');
+const url = require('url');
+const AWS = require('aws-sdk');
+const Sproxy = require('sproxydclient');
```

Comment:
We should pass the sproxydclient lib as a parameter of DataWrapper to avoid cycling dependencies

Reply:
See how we pass bucketclient in MetadataWrapper